contexts/NetworksContext.tsx

```diff
@@ -0,0 +1,46 @@
+import { createContext, useEffect, useState } from 'react';
+import Web3 from 'web3';
+
+export interface INetwork {
+  name: string;
+  rpc: string;
+  imgUrl?: string;
+}
+
+export interface INetworksContext {
+  network: INetwork;
+  setNetwork: (network: INetwork) => void;
+  web3: Web3;
+}
+
+export const NetworkContext = createContext<INetworksContext>({
+  network: { name: '', rpc: '' },
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  setNetwork: () => {},
```

Comment:
Could this return null or similar so we dont need the eslint ignore?
e.g. `() => null`